react/finger-exercises/redux/01 - actions and reducers/books/reducer.js

```diff
@@ -24,8 +25,18 @@ function reducer(state = initialState, action) {
     case actions.SET_FAVORITE_BOOK:
       return {
         ...state,
-        favoriteBook: action.payload.favoriteBook
+        favoriteBook: action.payload
       };
+
+    case actions.ADD_READ_BOOK:
+      return {...state}.readBooks.push(action.payload);
```

Comment:
Avoid using `push` since it mutates the array.
Also this won't work, push returns the new length of the array.